t/005-indexed-var-openresty-suites.t

```diff
@@ -346,17 +346,12 @@ variable not changeable
         }
     }
 --- config
-    location =/balancer {
-        set $port '';
-        set_by_lua_block $port {
-            local args, _ = ngx.req.get_uri_args()
-            local port = args['port']
-            return port
-        }
+    location ~ /balancer/(?<target_port>\d+) {
```

Comment:
Why do we change it? Could you give a explain?

Reply:
Because we didn't the `ndk` for OpenResty, so we can't use the `set_by_lua_block`.

Reply:
Got, thanks.